exalearn-ui/src/app/components/new-grammar/new-grammar.component.html

```diff
@@ -23,13 +23,24 @@
 			</mat-form-field>
 		</div>
 
+		<div class="reminder-answer">
+			<div *ngIf="!rightAnswer" class="reminder-text">
+				<p>Don't forget about right answer</p>
```

Comment:
As I know we need to move it to the constants and translate

Reply:
Really. My bad 🙈